wgpu-hal/src/vulkan/mod.rs

```diff
@@ -375,6 +380,37 @@ pub struct Surface {
     swapchain: RwLock<Option<Swapchain>>,
 }
 
+impl Surface {
+    /// Get the raw Vulkan swapchain associated with this surface.
+    ///
+    /// Returns `None` if the surface is not configured.
+    pub fn raw_swapchain(&self) -> Option<vk::SwapchainKHR> {
+        let read = self.swapchain.read();
+        read.as_ref().map(|it| it.raw)
+    }
+
+    /// Set the present timing information which will be used for the next presentation using `VK_GOOGLE_display_timing`.
```

Comment:
One more intradoc link?

Note that you didn't backtick the extension name below; can we settle on one and be consistent?